src/List.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+export default ({ ToDos }) => { // Export an annonymous function that takes in the ToDos we have and displays them in a list.
+
+    const clickHandler = (item) => {
+      item.completed = !item.completed;
+      const checkbox = document.getElementById(`${item.text}checkbox`);
+      const text = document.getElementById(`${item.text}`);
+      if (item.completed) {
+        checkbox.checked = true;
+        text.classList.add('crossed');
+        console.log(`'${item.text}' marked done.`);
+      } else {
+        checkbox.checked = false;
+        text.classList.remove('crossed');
+        console.log(`'${item.text}' marked not done.`)
+      }
+    };
+
+    return ( // Any time you want to use JavaScript inside JSX wrap in {}.
+      <ul>
+        {ToDos.map((ToDo, i) => { // For each value inside ToDos TODO why map instead of forEach?
```

Comment:
.map returns a copy of the original array where as forEach is better for looping over an array and mutating the original copy. It's a common pattern in react to use .map for all the things UI.